### PR TITLE
jupp: 31 -> 39

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/jupp/default.nix [hunk 1]
```diff
@@ -2,17 +2,18 @@
 , ncurses, gpm
 }:
 
+with stdenv.lib;
```
Comment (cole-h) [inline pkgs/applications/editors/jupp/default.nix:5]: Same comment as I made on your `mksh` PR: why was this moved? Doesn't seem necessary.

Comment (AndersonTorres) [reply pkgs/applications/editors/jupp/default.nix:5]: It makes the `stdenv.lib` functions available to all expression, not only `meta` part.

Comment (cole-h) [reply pkgs/applications/editors/jupp/default.nix:5]: Yes, but there's no need for `stdenv.lib` to be available anywhere else but the `meta` block, AFAICS.
